README.md

```diff
@@ -1,50 +1,50 @@
-# urlparse4
+# scurl
 
-`urlparse4` is a performance-focused replacement for Python's `urlparse` module, using C++ code from Chromium's own URL parser.
+`scurl` is a performance-focused replacement for Python's `urlparse` module, using C++ code from Chromium's own URL parser.
 
 Many credits go to [gurl-cython](https://github.com/Preetwinder/gurl-cython) for inspiration.
```

Comment:
would be nice to say it's based on urlparse4, and summarize the differences, since the name is changed now

Reply:
Sure! I will do that now :)